client/me/help/help-courses/course-videos.jsx

```diff
@@ -19,8 +19,14 @@ export default localize( ( props ) => {
 
 	return (
 		<div className="help-courses__course-videos">
-			<Card compact className="help-courses__course-videos-label">{ translate( 'Latest course' ) }</Card>
-			{ videos.map( ( video, key ) => <CourseVideo { ...video } key={ key }/> ) }
+			<Card compact className="help-courses__course-videos-label">
+				{ translate(
+					'Latest course',
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Latest course' )` *ES Score: 16.82*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/help-course-videos/add-seo-video)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).